package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@teamexos/apollo-cursor-pagination",
-  "version": "2.0.2",
+  "version": "2.0.1",
```

Comment:
This may cause an issue when redeploying. Can you push an update up to bump this?